src/langcheck/metrics/custom_text_quality.py

```diff
@@ -22,6 +25,9 @@ def custom_evaluator(
     score_map: dict[str, float],
     template_path: str,
     language: str,
+    *,
+    additional_params: dict[str, IndividualInputType] | None = None,
```

Comment:
Maybe "additional_inputs" would be better here?

Reply:
Makes sense, renamed!